webui-src/app/chat/chat.js

```diff
@@ -90,6 +153,20 @@ const ChatLobbyModel = {
             return this.users;
         });
     },
+    loadPublicLobby (currentlobbyid) {
+        console.info('loadPublicLobby ChatRoomsModel:',ChatRoomsModel);
+        this.setupAction= this.enterPublicLobby;
+        this.isSubscribed = false;
+        ChatRoomsModel.allRooms.forEach(it => {
+            if (it.lobby_id.xstr64 === currentlobbyid) {
+                this.currentLobby = it;
+                this.lobby_user = '???';
+                this.lobbyid = currentlobbyid;
+            }
+        })
+        this.users = [];
+        return this.users;
```

Comment:
Why set `users = []`? This will always return empty array.

Reply:
resetting the users; class is also used for subscribed lobbies and unsubsrcibed lobbies have no users to display

Reply:
In that case return isn't required.

Reply:
ah, now i've got it, it's not surrounded from loadLobbyDetails, where the return list is used